documents_operating_unit/security/documents_folder.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 RIL
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="documents_folder_rule" model="ir.rule">
+        <field name="name">Folders from allowed operating units</field>
+        <field name="model_id" ref="model_documents_folder" />
+        <field
+            name="domain_force"
+        >['|',('operating_unit_id','=',False),('operating_unit_id','in',user.operating_unit_ids.ids)]</field>
```

Comment:
Won't this have the same functional field issue as other operating unit modules. Don't we need assigned_operating_unit_ids?

Reply:
But this model is documents.folder, another one is in operating.unit itself. I'm a bit confused